lib/services/presets.go

```diff
@@ -85,6 +85,7 @@ func NewPresetEditorRole() types.Role {
 					types.NewRule(types.KindOktaImportRule, RW()),
 					types.NewRule(types.KindOktaAssignment, RW()),
 					types.NewRule(types.KindLock, RW()),
+					types.NewRule(types.KindIntegration, append(RW(), types.VerbUse)),
```

Comment:
I'm adding the VerbUse now to leverage the `defaultAllowRules`.

If we don't add it now, but only add it later, it won't do the migration.
Migrations only happens when the resource is not configured at all